vispy/visuals/text/text.py

```diff
@@ -463,7 +464,8 @@ def draw(self, transforms):
               transforms.framebuffer_to_render)
         px_scale = (tr.map((1, 0)) - tr.map((0, 1)))[:2]
         self._program.vert['transform'] = transforms.get_full_transform()
-        self._program.vert['text_scale'] = STTransform(scale=px_scale * n_pix)
+        self._text_scale.scale = px_scale * n_pix
+        self._program.vert['text_scale'] = self._text_scale
```

Comment:
we do expose the font size in points as a property, this his to do with the scaling done in the GLSL code (related to the font size in points, but not the same thing)